apps/privacy-panel/index.html

```diff
@@ -68,6 +68,7 @@
 
     <!-- Shared JavaScripts -->
     <script defer src="shared/js/l10n.js"></script>
+    <script src="shared/js/manifest_helper.js"></script>
```

Comment:
This can be commented like the script elements below it. Its just there for the build tools to find, it will actually get loaded by the alemeda loader (the define/require function calls)